en/admin/control.rst

```diff
@@ -2142,3 +2162,358 @@ The following shows how to use the CUBRID Manager (hereafter, CM) Administrator
+
+*   An example of configuring the **JVM_PATH** environment variable ::
+    
+    % JVM_PATH=/usr/java/jdk1.6.0_10/jre/lib/amd64/server/libjvm.so
+    % export JVM_PATH
+
+.. _cubrid-javasp-system-parameter:
+
+Java SP Server System Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following table shows the server paramters related to Java SP server available in the configuration file (**cubrid.conf**)
+
++-------------------------------------+--------+----------------+-----+-------+
+| Parameter Name                      | Type   | Default        | Min | Max   |
++-------------------------------------+--------+----------------+-----+-------+
+| java_stored_procedure               | bool   | no             |     |       |
++-------------------------------------+--------+----------------+-----+-------+
+| java_stored_procedure_port          | int    | 0              | 0   | 65535 |
++-------------------------------------+--------+----------------+-----+-------+
+| java_stored_procedure_jvm_options   | string |                |     |       |
++-------------------------------------+--------+----------------+-----+-------+
+
+For more details on these paramters, see :ref:`cubrid-conf`.
+
+.. _cubrid-javasp-service-util:
+
+Registering Java SP Server to cubrid service
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you register javasp to CUBRID service, you can use the utilities of **cubrid service** to start, stop or check all the registered javasp processes at once.
+
+- First, add **javasp** to the **service** parameter in the [**service**] section of the **cubrid.conf** file.
+- Second, To register the javasp server for a database, add the name of the database to the server parameter in the [**service**] section. Note that it shares the server parameter with the database server. a javasp server is dependent on the database server that has the same database name.
+- Finally, set **java_stored_procedure** as yes to enable starting the javasp server for the database.
+
+The following example shows how to register javasp server as service in the **cubrid.conf** file.
+Both *demodb* and *testdb* are present in the server property, but only demodb with **java_stored_procedure** set to yes is started by the **cubrid service start** command.
+
+::
+
+    # cubrid.conf
+
+    ...
+
+    [service]
+
+    ...
+
+    service=broker,server,javasp
+
+    # The list of database servers in all by 'cubrid service start' command.
+    # This property is effective only when the above 'service' property contains 'server' or 'javasp' keyword.
+    server=demodb,testdb
+
+    ...
+
+    [common]
+
+    ...
+
+    [@demodb]
+    java_stored_procedure=yes
+
+    [@testdb]
+    java_stored_procedure=no
+
+::
+
+    % cubrid service start
+    
+    @ cubrid master start
+    ++ cubrid master start: success
+    @ cubrid server start: demodb
+
+    This may take a long time depending on the amount of restore works to do.
+    CUBRID 10.2
```

Comment:
10.2 is right?

Reply:
It appears in all parts of the current manual. Updating the version in the examples needs to be done on a separate issue at once.